### ISSUE TITLE
validate submission by email

### ISSUE THREAD

**Issue by almet** [Kinto/formbuilder#175]

**Comment by almet** [comment]
If we decide to do this, it should be technically done via a kinto plugin rather than directly in the formbuilder itself.

**Comment by sahildua2305** [comment]
This is a good idea, but I think this will be a bit far fetched idea. We can certainly de-prioritize it to focus on the other important ideas. What do you think?

**Comment by almet** [comment]
I agree, the idea is not to put this prior to the other ones. We have milestones for the next items we want to work on. Here, the idea is to validate or not the idea, and then we'll priorize :-)

**Comment by almet** [comment]
I have created an "in the future"milestone for stuff that are interesting but not for just yet.
